modules/getting-started/con-audit-log-track-changes-catalog.adoc

```diff
@@ -0,0 +1,5 @@
+:_mod-docs-content-type: CONCEPT
+[id="con-audit-log-track-changes-catalog_{context}"]
+= Tracking changes in the {product-short} catalog database
```

Comment:
Headings that start with gerunds indicate procedural modules. I recommend changing the heading to:
```suggestion
= Changes in the {product-short} catalog database
```